### ISSUE TITLE
Impact of Giving page - stories

### ISSUE THREAD

**Issue by Mirandapow** [ucsc/site-giving#12]
I'm not sure the images to the right of the stories work well. Perhaps we try them on the left?

I also 
![Screen Shot 2020-03-06 at 10 03 57 AM](https://user-images.githubusercontent.com/60756584/76109677-d06e2f00-5f91-11ea-96c5-7e8569848830.png)
think all of the images should be consistent in size - all square.

**Comment by jhillard-ucsc** [comment]
It would be great if there were a thumbnail tool.

**Comment by Herm71** [comment]
- Per our image size discussion this morning, we'll get them uniformized with respect to the heights -- aspect ratio, etc.
- As for the left/right issue, I originally had them on the left but when Rob and I discussed it, we decided to put them on the right to account for a use-case in which there was no featured image. If the images were on the left and we had an article that didn't have a featured image, it would look funny.
